src/controllers/__tests__/profileController.test.ts

```diff
@@ -47,8 +59,32 @@ jest.mock("../../services/profileService", () => ({
     updateProfile: mockUpdateProfile
   }))
 }));
+const mockSendEmail = jest.fn();
+jest.mock("../../services/emailService", () => ({
+  default: jest.fn().mockImplementation(() => ({
+    send: mockSendEmail
+  }))
+}));
 
-const controller = new ProfileController(new ProfileService());
+// tslint:disable-next-line:no-let
+let controller: ProfileController;
+
+beforeAll(async () => {
+  const testEmailAccount = await nodemailer.createTestAccount();
+  controller = new ProfileController(
```

Comment:
another way is to have a `getProfileController()` factory method to use inside the tests so you don't have to share a global variable